server/commands/teamCreator.ts

```diff
@@ -93,6 +93,7 @@ async function teamCreator({
     });
   }
 
+  // eslint-disable-next-line   @typescript-eslint/no-non-null-assertion
```

Comment:
I think maybe your editor is applying additional eslint rules?

Reply:
It would be nice if we could have the `any` rule but because the codebase was relatively recently converted from Flow there's still some work to do there 😅